Swift/Controllers/XMLConsoleController.cpp

```diff
@@ -41,5 +45,11 @@ void XMLConsoleController::handleDataWritten(const SafeByteArray& data) {
         xmlConsoleWidget->handleDataWritten(data);
     }
 }
-
+void XMLConsoleController::setClient(std::shared_ptr<Client> client_)
+{
+    this->client_ = client_;
+}
+void XMLConsoleController::sendXML(std::string data){
+    client_->sendData(data);
 }
+}
```

Comment:
There is a newline missing at the end of the file. While it's not required for C++11, currently all files in Swift have newlines at the end of the source files and our CI enforces this.